integration/integrate-using-short-lived-tokens.adoc

```diff
@@ -0,0 +1,46 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="integrate-using-short-lived-tokens"]
+= Integrating {product-title-short} using short-lived tokens
+include::modules/common-attributes.adoc[]
+:context: integrate-using-short-lived-tokens
+
+toc::[]
+
+[role="_abstract"]
+{rh-rhacs-first} integrations are able to authenticate against selected cloud provider APIs with short-lived tokens. The following cloud provider integrations are supported:
+
+* Amazon Web Services (AWS) using the Secure Token Service (STS)
+* Google Cloud Platform (GCP) using workload identity federation
+
+Short-lived token integrations are supported when {product-title-short} is installed on the following platforms:
+
+* Elastic Kubernetes Service (EKS) on AWS
+* Google Kubernetes Engine (GKE) on GCP
+* {ocp} based platforms
+
+To activate short-lived authentication, one must first establish a trust relationship between the {product-title-short} cluster and the cloud provider. For EKS and GKE clusters, this is achieved natively through the cloud provider metadata service. For {ocp} clusters, a publicly available OIDC provider bucket containing the {ocp} service account signer key is required.
```

Comment:
This should be Kubernetes not RHACS cluster, it might be confusing for people reading this.